client/state/sites/reducer.js

```diff
@@ -40,6 +41,14 @@ const VALID_SITE_KEYS = Object.keys( sitesSchema.patternProperties[ '^\\d+$' ].p
  */
 export function items( state = {}, action ) {
 	switch ( action.type ) {
+		case WORDADS_SITE_APPROVE_REQUEST_SUCCESS:
```

Comment:
We cannot move this particular piece of information anywhere, because this is supposed to be here in the sites-list endpoint.
It was here all along.
I'm just setting it here manually, because I dont want to reload the sites endpoint, but `options.wordads` is an information that was always in API response.